.github/actions/test-khiops-install/action.yml

```
@@ -194,12 +204,17 @@ runs:
           echo "::error::This action expects CONDA_ENV_BIN_DIR to be defined"
           false
         fi
-        $KHIOPS_ENV_DIR/khiops_coclustering -s
-        $KHIOPS_ENV_DIR/khiops_coclustering -v &> output
+        $KHIOPS_ENV_DIR/khiops_coclustering -s | tee output_s
```

Idem above, `| tee output_file` is not identical to `&> output_file` because previous command's standard error is lost.

---

done